internal/commands/user_test.go

```diff
@@ -107,3 +109,145 @@ func buildSlackMock() *slack.User {
 		},
 	}
 }
+
+func TestGetStringInt64(test *testing.T) {
```

Comment:
I think this test was created in the wrong file. Of course, all those functions are inside the same package, but in my opinion, this new test should be in a file named `util_test.go`

Reply:
It's true. Fixed it.